src/main/java/com/laytonsmith/core/constructs/CClassType.java

```diff
@@ -454,8 +455,8 @@ public boolean unsafeIsExtendedBy(CClassType checkClass) {
 	 *
 	 * @return
 	 */
-	protected Set<CClassType> getTypes() {
-		Set<CClassType> t = new HashSet<>();
+	protected TreeSet<CClassType> getTypes() {
```

Comment:
[Dependency Inversion](https://en.wikipedia.org/wiki/Dependency_inversion_principle) is the preferred approach here, so still want to return Set, as well as use Set on the lhs of the assignment.

Reply:
Oh right. I usually do that but I forget sometimes.